settings.gradle.kts

```diff
@@ -4,6 +4,7 @@
 
 pluginManagement {
     repositories {
+        mavenLocal()
```

Comment:
For what this added?

Reply:
For changes testing without version release.